src/js/components/TaskView.js

```diff
@@ -60,6 +61,24 @@ class TaskView extends mixin(SaveStateMixin, StoreMixin) {
     super.componentWillMount();
   }
 
+  componentWillReceiveProps(nextProps) {
+    let {filterByStatus, searchString} = this.state;
+    let tasks = this.getFilteredTasks(filterByStatus, nextProps.tasks, searchString);
+    let prevTasks = this.getFilteredTasks(filterByStatus, this.props.tasks, searchString);
+
+    if (!deepEqual(tasks, prevTasks)) {
```

Comment:
There isn't a super convenient way to use fixtures with TaskView so I made some observations on the OrganizationTab, and qualitatively we will be running into table rendering performance snags in scrolling before sorting becomes an issue